--- -	2026-04-26 16:06:36.100166223 +0900
+++ ggml/src/ggml-cuda/ggml-cuda.cu	2026-04-26 15:16:47.884559963 +0900
@@ -4957,7 +4957,18 @@
             {
                 return (op->type == GGML_TYPE_F32 || op->type == GGML_TYPE_F16 || op->type == GGML_TYPE_BF16 ||
                        op->type == GGML_TYPE_Q4_0 || op->type == GGML_TYPE_Q4_1 || op->type == GGML_TYPE_Q5_0 ||
-                       op->type == GGML_TYPE_Q5_1 || op->type == GGML_TYPE_Q8_0 || op->type == GGML_TYPE_IQ4_NL) &&
+                       op->type == GGML_TYPE_Q5_1 || op->type == GGML_TYPE_Q8_0 || op->type == GGML_TYPE_IQ4_NL ||
+                       op->type == GGML_TYPE_TBQ3_0 || op->type == GGML_TYPE_TBQ4_0 ||
+                       op->type == GGML_TYPE_TBQP3_0 || op->type == GGML_TYPE_TBQP4_0 ||
+                       op->type == GGML_TYPE_TBQ3_1 || op->type == GGML_TYPE_TBQ4_1 ||
+                       op->type == GGML_TYPE_TBQP3_1 || op->type == GGML_TYPE_TBQP4_1 ||
+                       op->type == GGML_TYPE_TBQ3_2 || op->type == GGML_TYPE_TBQ4_2 ||
+                       op->type == GGML_TYPE_TBQP3_2 || op->type == GGML_TYPE_TBQP4_2 ||
+                       op->type == GGML_TYPE_TBQ3_3 || op->type == GGML_TYPE_TBQ4_3 ||
+                       op->type == GGML_TYPE_TBQP3_3 || op->type == GGML_TYPE_TBQP4_3 ||
+                       op->type == GGML_TYPE_TBQ3_4 || op->type == GGML_TYPE_TBQ4_4 ||
+                       op->type == GGML_TYPE_TBQP3_4 || op->type == GGML_TYPE_TBQP4_4 ||
+                       op->type == GGML_TYPE_AMX3_1 || op->type == GGML_TYPE_AMXV3_1) &&
                        op->src[0]->type == GGML_TYPE_F32 &&
                        (op->src[1]->type == GGML_TYPE_I64 || op->src[1]->type == GGML_TYPE_I32);
             } break;
