diff --git a/ggml/src/ggml-cpu/ops.cpp b/ggml/src/ggml-cpu/ops.cpp
index a9bc21da6..a63e5017b 100644
--- a/ggml/src/ggml-cpu/ops.cpp
+++ b/ggml/src/ggml-cpu/ops.cpp
@@ -5591,6 +5591,28 @@ void ggml_compute_forward_clamp(
         case GGML_TYPE_I32:
         case GGML_TYPE_I64:
         case GGML_TYPE_F64:
+        case GGML_TYPE_TBQ3_0:
+        case GGML_TYPE_TBQ4_0:
+        case GGML_TYPE_TBQP3_0:
+        case GGML_TYPE_TBQP4_0:
+        case GGML_TYPE_TBQ3_1:
+        case GGML_TYPE_TBQ4_1:
+        case GGML_TYPE_TBQP3_1:
+        case GGML_TYPE_TBQP4_1:
+        case GGML_TYPE_TBQ3_2:
+        case GGML_TYPE_TBQ4_2:
+        case GGML_TYPE_TBQP3_2:
+        case GGML_TYPE_TBQP4_2:
+        case GGML_TYPE_TBQ3_3:
+        case GGML_TYPE_TBQ4_3:
+        case GGML_TYPE_TBQP3_3:
+        case GGML_TYPE_TBQP4_3:
+        case GGML_TYPE_TBQ3_4:
+        case GGML_TYPE_TBQ4_4:
+        case GGML_TYPE_TBQP3_4:
+        case GGML_TYPE_TBQP4_4:
+        case GGML_TYPE_AMX3_1:
+        case GGML_TYPE_AMXV3_1:
         case GGML_TYPE_COUNT:
             {
                 GGML_ABORT("fatal error");
