diff --git a/ggml/src/ggml-cuda/ggml-cuda.cu b/ggml/src/ggml-cuda/ggml-cuda.cu
index 415a9f90d..e9630229d 100644
--- a/ggml/src/ggml-cuda/ggml-cuda.cu
+++ b/ggml/src/ggml-cuda/ggml-cuda.cu
@@ -4843,7 +4843,9 @@ static bool ggml_backend_cuda_device_supports_op(ggml_backend_dev_t dev, const g
                        op->type == GGML_TYPE_TBQ3_1 || op->type == GGML_TYPE_TBQ4_1 ||
                        op->type == GGML_TYPE_TBQP3_1 || op->type == GGML_TYPE_TBQP4_1 ||
                        op->type == GGML_TYPE_TBQ3_2 || op->type == GGML_TYPE_TBQ4_2 ||
-                       op->type == GGML_TYPE_TBQP3_2 || op->type == GGML_TYPE_TBQP4_2) &&
+                       op->type == GGML_TYPE_TBQP3_2 || op->type == GGML_TYPE_TBQP4_2 ||
+                       op->type == GGML_TYPE_TBQ3_3 || op->type == GGML_TYPE_TBQ4_3 ||
+                       op->type == GGML_TYPE_TBQP3_3 || op->type == GGML_TYPE_TBQP4_3) &&
                        op->src[0]->type == GGML_TYPE_F32 &&
                        (op->src[1]->type == GGML_TYPE_I64 || op->src[1]->type == GGML_TYPE_I32);
             } break;
