diff --git a/ggml/src/ggml-cuda/ggml-cuda.cu b/ggml/src/ggml-cuda/ggml-cuda.cu
index 185956317..073044359 100644
--- a/ggml/src/ggml-cuda/ggml-cuda.cu
+++ b/ggml/src/ggml-cuda/ggml-cuda.cu
@@ -4927,7 +4927,18 @@ static bool ggml_backend_cuda_device_supports_op(ggml_backend_dev_t dev, const g
             {
                 return (op->type == GGML_TYPE_F32 || op->type == GGML_TYPE_F16 || op->type == GGML_TYPE_BF16 ||
                        op->type == GGML_TYPE_Q4_0 || op->type == GGML_TYPE_Q4_1 || op->type == GGML_TYPE_Q5_0 ||
-                       op->type == GGML_TYPE_Q5_1 || op->type == GGML_TYPE_Q8_0 || op->type == GGML_TYPE_IQ4_NL) &&
+                       op->type == GGML_TYPE_Q5_1 || op->type == GGML_TYPE_Q8_0 || op->type == GGML_TYPE_IQ4_NL ||
+                       op->type == GGML_TYPE_TBQ3_0 || op->type == GGML_TYPE_TBQ4_0 ||
+                       op->type == GGML_TYPE_TBQP3_0 || op->type == GGML_TYPE_TBQP4_0 ||
+                       op->type == GGML_TYPE_TBQ3_1 || op->type == GGML_TYPE_TBQ4_1 ||
+                       op->type == GGML_TYPE_TBQP3_1 || op->type == GGML_TYPE_TBQP4_1 ||
+                       op->type == GGML_TYPE_TBQ3_2 || op->type == GGML_TYPE_TBQ4_2 ||
+                       op->type == GGML_TYPE_TBQP3_2 || op->type == GGML_TYPE_TBQP4_2 ||
+                       op->type == GGML_TYPE_TBQ3_3 || op->type == GGML_TYPE_TBQ4_3 ||
+                       op->type == GGML_TYPE_TBQP3_3 || op->type == GGML_TYPE_TBQP4_3 ||
+                       op->type == GGML_TYPE_TBQ3_4 || op->type == GGML_TYPE_TBQ4_4 ||
+                       op->type == GGML_TYPE_TBQP3_4 || op->type == GGML_TYPE_TBQP4_4 ||
+                       op->type == GGML_TYPE_AMX3_1 || op->type == GGML_TYPE_AMXV3_1) &&
                        op->src[0]->type == GGML_TYPE_F32 &&
                        (op->src[1]->type == GGML_TYPE_I64 || op->src[1]->type == GGML_TYPE_I32);
             } break;
